src/components/Course.jsx

```diff
@@ -1,43 +1,45 @@
-// @flow
-
 import React from 'react';
-import { observer } from 'mobx-react';
+import { inject, Observer } from 'mobx-react';
 import { Draggable } from 'react-beautiful-dnd';
 import '../styles/objects.Course.scss';
 import '../styles/utilities.shadow.scss';
 import '../styles/utilities.center.scss';
 
-export const Course = observer( ({colorScheme, course}) => (
+export const Course = inject('store')(({ store, course }) => (
```

Comment:
ahhhh so THAT'S how you do this

Reply:
Yeah, I try not to use provider / inject until it gets gross to not use it, cuz it kinda breaks some principals- but we have a good use case here, so it was time.